src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2896,6 +2925,45 @@ because xbuild doesn't support framework reference assemblies.
     UncompressedFileExtensions="$(AndroidStoreUncompressedFileExtensions)">
     <Output TaskParameter="OutputFiles" ItemName="ApkFiles" />
   </BuildApk>
+  <BuildBaseAppBundle
+    Condition=" '$(AndroidPackageFormat)' == 'aab' "
+    AndroidNdkDirectory="$(_AndroidNdkDirectory)"
+    ApkInputPath="$(_PackagedResources)"
+    ApkOutputPath="$(_BaseZipIntermediate)"
+    BundleAssemblies="$(BundleAssemblies)"
+    BundleNativeLibraries="$(_BundleResultNativeLibraries)"
+    EmbedAssemblies="$(EmbedAssembliesIntoApk)"
+    ResolvedUserAssemblies="@(_ResolvedUserAssemblies);@(_AndroidResolvedSatellitePaths)"
+    ResolvedFrameworkAssemblies="@(_ShrunkFrameworkAssemblies)"
+    NativeLibraries="@(AndroidNativeLibrary)"
+    AdditionalNativeLibraryReferences="@(_AdditionalNativeLibraryReferences)"
+    EmbeddedNativeLibraryAssemblies="$(OutDir)$(TargetFileName);@(ReferencePath);@(ReferenceDependencyPaths)"
+    DalvikClasses="@(_DexFile)"
+    SupportedAbis="$(_BuildTargetAbis)"
+    CreatePackagePerAbi="False"
+    UseSharedRuntime="$(AndroidUseSharedRuntime)"
+    Debug="$(AndroidIncludeDebugSymbols)"
+    PreferNativeLibrariesWithDebugSymbols="$(AndroidPreferNativeLibrariesWithDebugSymbols)"
+    TypeMappings="$(_AndroidTypeMappingJavaToManaged);$(_AndroidTypeMappingManagedToJava)"
+    JavaSourceFiles="@(AndroidJavaSource)"
+    JavaLibraries="@(AndroidJavaLibrary)"
+    AndroidSequencePointsMode="$(_SequencePointsMode)"
+    LibraryProjectJars="@(ExtractedJarImports)"
+    AndroidEmbedProfilers="$(AndroidEmbedProfilers)"
+    TlsProvider="$(AndroidTlsProvider)"
+    UncompressedFileExtensions="$(AndroidStoreUncompressedFileExtensions)">
+    <Output TaskParameter="OutputFiles" ItemName="BaseZipFile" />
+  </BuildBaseAppBundle>
+  <BuildAppBundle
+    Condition=" '$(AndroidPackageFormat)' == 'aab' "
+    ToolPath="$(JavaToolPath)"
+    JavaMaximumHeapSize="$(JavaMaximumHeapSize)"
+    JavaOptions="$(JavaOptions)"
+    JarPath="$(AndroidBundleToolJarPath)"
+    BaseZip="$(_BaseZipIntermediate)"
+    Output="$(_AppBundleIntermediate)"
+    UncompressedFileExtensions="$(AndroidStoreUncompressedFileExtensions)"
+  />
```

Comment:
I could move these to the new file and use `<CallTarget/>`, but it wouldn't be able to use this:
```
<Output TaskParameter="NativeLibrariesReferences" ItemName="_AdditionalNativeLibraryReferences" />
```